src/main/java/de/tum/in/www1/artemis/service/GroupNotificationService.java

```diff
@@ -120,6 +120,16 @@ public void notifyInstructorGroupAboutDuplicateTestCasesForExercise(Exercise exe
         saveAndSend(createNotification(exercise, null, GroupNotificationType.INSTRUCTOR, NotificationType.DUPLICATE_TEST_CASE, notificationText));
     }
 
+    /**
+     * Notify instructor groups about invalid submissions
+     *
+     * @param exercise         that has been affected
+     * @param notificationText that should be displayed
+     */
+    public void notifyInstructorGroupAboutInvalidSubmissionsForExercise(Exercise exercise, String notificationText) {
+        saveAndSend(createNotification(exercise, null, GroupNotificationType.INSTRUCTOR, NotificationType.ILLEGAL_SUBMISSION, notificationText));
```

Comment:
Can we add the student name here and include this name in the notification text?

Reply:
done in 2aef655083373af302f175c973b2e466b5c230a1